config.py

```diff
@@ -83,9 +83,9 @@ class LocalConfig(BaseConfig):
     DEBUG = True
 
     # Using a local postgre database
```

Comment:
I see no purpose of changes in  this file for this issue🤔

Reply:
> I see no purpose of changes in this file for this issue🤔

It's probably because the contributor is using windows and maya uses windows. Not the contributors fault. This should be done with environment variables.